app/serializers/trainee_serializer/v01.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+module TraineeSerializer
+  class V01
```

Comment:
Scoping this by version is a good idea. It means we can avoid trying to name this anything hesa specific, once it's dropped the newer versions will have the register specific data